### PR TITLE
PXP-644 feat: add application init command

### PR DESCRIPTION
**Pull request by mfauzaan**
## Summary
Build the workflow for the wukong application init command to satisfy the specifications. Details about the command inputs and outputs can be found [here](https://mindvalley.atlassian.net/wiki/spaces/PXP/pages/762937345/RFC18+-+Per-Application+Initialisation+Command).

Screenshot 2024-02-02 at 11 35 23 AM

Ticket: https://mindvalley.atlassian.net/browse/PXP-644

### Added 

- [x] Added `wukong application init` command

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/192#" title="Author self-approved">mfauzaan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: cli/src/commands/application/init.rs
Comment (onimsha) [inline cli/src/commands/application/init.rs:27]:
```suggestion
    let excluded_workflows = inquire::MultiSelect::new("Workflows to exclude from the Wukong CLI & TUI", workflows)
```

File: cli/src/commands/application/init.rs
Comment (onimsha) [inline cli/src/commands/application/init.rs:37]:
```suggestion
    let addons = ["Elixir Livebook"];
```

File: cli/src/commands/application/init.rs
Comment (onimsha) [inline cli/src/commands/application/init.rs:47]:
```suggestion
    let configure_staging_namespace = inquire::Confirm::new("Do you want to configure the staging namespace?")
```

File: cli/src/commands/application/init.rs
Comment (onimsha) [inline cli/src/commands/application/init.rs:83]:
```suggestion
        inquire::Editor::new("Do you want to review the .wukong.toml file before writing to disk ?")
```

File: cli/src/commands/application/init.rs
Comment (onimsha) [inline cli/src/commands/application/init.rs:129]:
```suggestion
    let base_replica = CustomType::<u32>::new("Number of replicas")
```

File: cli/src/commands/application/init.rs
Comment (onimsha) [inline cli/src/commands/application/init.rs:147]:
```suggestion
    let cloudsql_project_id = inquire::Text::new("Google Project of your CloudSQL instance(s)")
```

**Review by onimsha — APPROVED** [review]

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/192#" title="Author self-approved">mfauzaan</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/192#pullrequestreview-1861853578" title="Approved">onimsha</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
